src/main/resources/views/CommercialVehicles/RWT_DATA.hbs

```diff
@@ -0,0 +1,71 @@
B6uj+cjkTOQ7NTjOVwMyt9Rp/u7GTD1h19pk+PlJ1y+KZwrBSuP+yd3ZKbMAxG7VEgJCQTSAh/Ad7/MXvR3bjhw8Iy3RbP+Fy2y5LM6mAJy7aHDhj2iU2TK1txmFgyTB2jg4n5umUS/Q51wEXcaFgNMypfEM6WV5gure66lsWV5LtHrgMmRXSyPQaHlVRpUkht1aHUXJHepxCQoENm+RaYLZ3J2HXmR7grFtN8atmryE7ZoMPVGpwjt8NMawLtuWiZTYcebLCJ1KEOaANK1oKzTHSP8x94zr4WoOO+Crtngw6ptVmvJSYvyCFTYt7oEOqH8YRP9GRRh2TlqV2BehDceJnxhdjpePPxGxXZK/46tGQfAjp75E60ur3AuKwDrcZTaW4LOtgtui6kdpf1xtzCvFp1Kqa7WEjvH38dLkxYP+wjQLXet9PTkg4lRC2Sw20nKNuB80ItTeAQQvPr+I6PuMlIAPjrQCYjlwwBmkkpUBmNWxToQtkw+XuKOtSr9tUKr8qUlWyuzMDVNg+TfEV2i1gHbEhl6wo6W0QhLqoJR4cXQaxx98XAvigbqEMCgwNb88O94qRDoLA6+Lfumzx6hEeqQ1TXqEMJyTwraQMh+lI2DqADOWU2zTcu1Y2p1yP7xVeHiS9sTYdDbcnuE8WQoQ5Hp6UCr7mkk/kc7jr0MMHtxsPuehUHhxDw1WFcSSeOlog6OAVaATpgzPKhnc51SNev6eD20uC1Xxf3nwoCXx3WUvJm+eyTJ0GcszMIxJcOSGrL53PB6FD5Bu9gq8B7s8NCZMfIdMBop3b1RKAe2pncH/IED+yhaBmK3ORUnzrcBDrkHjrwxfQYc6UgEOggW+IzmoksQcs/bPyKow2x6G8auQ4dZHSjEkKWvqS4ZUAY+OnwNGF3sEB6yZiHY1wkcx0yLaSa6XAXjA7kvYjztmxfH7v3wsBPh0o7gflU4qjDZa7DqIUc/XXoCaaXNxbTl5grhYGfDtLIRB2Sn9ZhkOvQmWe5tw7p4vvguGVAIHjpUGoZgehQgw5PJaVYGv7OcXAIBC8djlrG4z8kS+nf0EHJoYVrk9i9Fwg+OrwO0tAU63Cc63D5Bzp0UDsUSsxt4Vqz+C+yb3x0eGghVIAOg2Kp3zrA0ruHG9dNtYNcB2zkjrlSePjo0EnnAHSCXT3CabhS/OZ+05slvEheTMNQd1ORneOhw8l0q7Ysd+MDdrry6RjZZ6V/XgdFW9pPW+jHiINDMAh0gITncHZ+H3udq0QTH1LYs/QPdYCiRwB2602xey8YBDpAN1LuPls3QF94xl7IdLT+Ax0St56lqf3iuVxMf+dK8fDIYJDrkGnnzHoiGAwOTjWAKU4wJf9BHYQpXU1flMsdGdeYK4WGXIdU8Fa+hj7wHNZJ8C1RaGHz8zq8wHjR6ob04ysWsXsvHMQ6tOQwsYyP2Q7GlpPLm1zCuM4VR/X4TbZFhx6SPNE+Yu2HJsN7cWpk90h1MPNhdBJN2F1xjw3JxjKmZOmc1g3lXjrgqOaSNaY2T6qYKwWGWAcSLY3JYVGcfvPk4szoAPVt5bJ1wH2TDg232I9rA8cG1vbdqBLZP1Id7rJzoxs932Usw0ZX7mCFhea4TpDOe+rQwwdAWuZHtFneMMTBISCkOqSwGzGPCe3LPNjpzo0ouKS6A4/s9qXbdFADjGoANzlRm/Lq/UORABDoAKm/CwnsgneBneKxB86wONJUq0VHs1GHl33bb/gaJ2biXp3ilgEhIdRhNKWxtKNDN5AL1f2aDcRMciMp5FMyHfBX4ZV4vgM3dDzTmCuFhFAHCALBFTk2xHYFjiYfkEWtUbEHTmebdXhpw52t9ktu7DjGA3+CQqZDw9fBK7lPj+f70O2zUP/+r2GmA1QVNWQfJR5rJW/wNozMwRCvo36TsruCxy0DwkKmQw7Hpa1SELyLeh60gS7lV6y0N+PJREs69NpAw7VXhjIneIXrqwMelUjVn7/ycuAD3SgfD/wJDJEOTxL8fTEwasv55dSlx+HjwObRcq50cfi4Lh+zpmyb++1IsBZ1U7Jkahxj33ht2lPzSOqPezVOy8nvKhIEIh0qyN0dyDTOY2ekOQYFOhiR5sC/JGqLDuiegeBeFVszxU3sf7V3bzuKwzAYgP3LTbNpQVMo03N33/8td9hp6ZaeOczIjL97FCHFKDiOLc2mcECL329qUWaptWdMy7sPMV05elhg7izh60TLa81nmHXgjzRbwiHFTU/SqrFfSTf/6D+a7GzsMK++u6K1UyaYdVoYhqIDf6TZEg4WuKUAJx3dP4VljOHd/7uYaaCwmJHeVeA9YBjTHM3Rs5JAXThsKE/ljePYx7OSh4pxjX1HZ25updTHON4V9NhwIFdhHNfLvwLaMkCaDeFwwo3dLAwm2t9nBgx02Dr6dJzPYKWhN7ZBy/uf/wwd7HAtxltAPToN7iX8aRxp0T5uZbRJFF+kdMXFpoL3IQ/rtBhc3iU0odzbnBkNZn93oDFB3IjWPhkaCo4m4W4tz68dDenAH/Us1ZpC0NKlTZBFBT1ZmbVrZSvXCqAtA9RjePJTlEbPSmqz+TpyQ2LpwB/1IL/kVzc4NAJSas5PSMok8r+Cer6TbQQ0KYL8J5Xd0wulJu1W3OmGEP/XQafBqTX2y1vdocFy54OgUZFS07LF/wVFLv+s5HTgj3pMA/AQ8gdtsp6V1Cr1QulTCPnVDVZ+olh9jZLRSjK65hJAfHVD/QKJYvVFalxw2N/ykWFA6nWuNWnxWbgOzbKq1RKgwzY+REERBNGx9hmQe1QygOdb38MFk1JLSlzhf9CTiytuMLjQTpRqvRIjpEdDGw5yT3vqmxQh+l5hJxl8kHzaU99mjxGyLxyMNqJUN4sTTDiRSP1wqDQa1CaHXc7oYxixtw0WgKZY1R1+H99M5eMsr8zbPiK53mNb4azaiQ3pH+4vSLtZnTv7G8gAAAAASUVORK5CYII=" alt="DVSA Crown" id="logo"/>
```

Comment:
Well, I guess that's one way to do it, but do you then need the "background-image" url css? Just causes confusion for those who come after

Reply:
updated css